theme/inc/block-editor.php

```diff
@@ -114,6 +114,8 @@ function enqueue_block_editor_assets() {
 		$version,
 		false
 	);
+
+	wp_add_inline_style( 'material-google-fonts', \MaterialDesign\Theme\Customizer\get_css_vars() );
```

Comment:
Currently `material-block-editor-css-theme` handle is not working, we are doing this to pull the variable. `--mdc-theme-header`